---
permalink: /
title: "Biography"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a computational biologist who enjoys exploring cell signalling through data-driven mathematical modeling. Recently graduated from the joint Master‚Äôs programme in Molecular Techniques in Life Science, I am a PhD candidate in [Evangelia Petsalaki's research group](https://www.ebi.ac.uk/research/petsalaki/) (Whole Cell Signalling) at the European Bioinformatics Institute and the University of Cambridge as part of the [EMBL International PhD Programme](https://www.embl.org/about/info/embl-international-phd-programme/).

During my studies in Heidelberg and Stockholm, I specialized in systems biology by attending several classes dealing with programming languages, mathematical modeling/computational pathway simulations, and statistical methods, which provided me insight into data science and the complexity of dynamic processes in living cells. I am especially interested in understanding the dynamic interplay of systems components and how information is specifically processed and linked to cell-fate decisions despite presence of (biochemical) noise.

As an active member of the preLights community, I select and highlight interesting preprints and interact with their authors. Launched in 2018, preLights is a community supported by The Company of Biologists, which provides a platform for highlighting and discussing new preprints across the biological sciences. My preLight-posts can be found [here](https://prelights.biologists.com/profiles/bdmaier/).

#### Scientific Programming Languages ####
Python (advanced), R (advanced), C++ (intermediate), Julia (beginner), Matlab (beginner) 

#### Cell Modelling and Simulation Tools ####
COPASI (advanced), COBRApy (intermediate), VCell (intermediate), NFsim (intermediate), PhysiBoSS/MaBoSS/PhysiCell (intermediate), Kappa (beginner)

#### Other Frameworks ####
Containerisation (Singularity & Docker), workflow management systems (Nextflow & SnakeMake), scripting (Shell/Bash, LaTex), version control systems (GitHub & GitLab), high performance computing (Codon & SLURM)

#### Skills ####
Mathematical Mechanistic Modeling (Rule-based, ODE-based, logical | Whole-Cell, Genome-Scale, Detailed | Metabolic, Signalling, Population), Machine Learning, Genetic Algorithms, Network Analysis, Data Analysis, Statistical Modeling and Algorithms, Sequence Analysis, Seeding Algorithms

#### Languages ####
German (C2), English (C2), Swedish (B2), French (A2), Spanish (A2)


News
======

#### 2024 ####
- üó£Ô∏è November: Presented "SELPHI2: Data-driven extraction of human kinase-substrate relationships from omics datasets" at SymBLS.
- üå± November: [Workshop Report](https://doi.org/10.1101/gr.277645.123) out in Science Editor highlighting how publishers and early-career researchers are working together and guiding initiatives.
- üå± September: [Preprint](https://www.biorxiv.org/content/10.1101/2022.01.15.476449) out in bioRxiv introducing a machine learning-based model to derive a probabilistic kinase-substrate network from omics datasets and an associated [webserver](https://selphi2.com/).
- ‚ô¶Ô∏è February: Invited by the Company of Biologist to attend the 9th Researcher to Reader Conference in London, UK as PreLight representative and represent early career researchers in the Workshop: Publishers and ECRs Working Together.

#### 2023 ####
- üó£Ô∏è December: I was interviewed by Reinier Pros√©e, the Community Manager of preLights, where we discussed my journey into science, the role of preprints, and my current research interests. You can read the full interview [here](https://prelights.biologists.com/news/meet-the-prelighters-a-catch-up-with-benjamin-dominik-maier/).
- üå± May: [Paper](https://doi.org/10.1101/gr.277645.123) out in Genome Research showing that entropy predicts fuzzy-seed sensitivity.
- üó£Ô∏è April: Presented "Entropy predicts sensitivity of pseudo-random seeds" at RECOMB
- ‚ô¶Ô∏è March: Awarded a travel fellowship from PerMedCoE to attend the PerMedCoE summer school: from pathway modelling tools to cell-level simulations in Seva, Spain
- ‚ô¶Ô∏è March: Awarded a travel fellowship from RECOMB to present my work at RECOMB in Istanbul, Turkey
- ‚ô¶Ô∏è January: Awarded a travel fellowship from Churchill College, Cambrige University, UK to present my work at RECOMB in Istanbul, Turkey

#### 2022 ####
- üå± October: [Preprint](https://www.biorxiv.org/content/10.1101/2022.10.13.512198) out showing that entropy predicts fuzzy-seed sensitivity
- üå± October: [Paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010623) out in PLOS Computational Biology on stochastic dynamics of type-I interferon responses.
- üéì June: Graduated with a joint M.Sc. in Molecular Techniques in Life Science from Karolinska Institute, KTH Royal Institute of Technology and Stockholm University, Sweden
- ‚ô¶Ô∏è February: Admission to the EMBL International PhD programme (acceptance rate ~ 2%)

#### 2021 ####

- üó£ September: Featured talk on the "quantitative interplay of chromatin organization and gene co-expression at the single-cell level" at EpiChrom
- ‚ô¶Ô∏è March: Awarded with the SciLifeLab summer internship fellowship to "predict nuclear 3D organization from single-cell gene expression" in the Pelechano Lab (Karolinska Institute)

#### 2020 and before ####

- üéì June 2020: Graduated with a B.Sc. in Biological Science from Heidelberg University, Germany
- ‚ô¶Ô∏è January 2020: Awarded with a graduate fellowship by the German Academic Scholarship Foundation
- ‚ô¶Ô∏è January 2018: Awarded with a undergraduate fellowship by the German Academic Scholarship Foundation
- ‚ô¶Ô∏è June 2017: Karl-von-Frisch-Award for outstanding high school graduate achievements in the subject of biology.