---
layout: archive
title: "PreLight Posts"
permalink: /prelights/
author_profile: true
---

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

preLights is the preprint highlights service run by the biological community and supported by The Company of Biologists. As a preLighter, I select, highlight and comment on preprints that I feel are of particular interest to the biological community. You’ll find a summary of each preprint, the reasons it was selected and my thoughts on its significance alongside comments from the preprints's authors addressing the questions I have raised.I’d really welcome you to check out my preLight post and make sure to leave your thoughts and comments. You can find a complete list of my preLight posts on my <a href="https://prelights.biologists.com/profiles/bdmaier/">preLights profile</a>.

<h2>Preprints I featured as a PreLight</h2>

#### 2024 ####

<p>Iva Veseli, Zachary S. Cooper, Michelle A. DeMers, et al. Digital Microbe: A Genome-Informed Data Integration Framework for Collaborative Research on Emerging Model Organisms. <em>bioRxiv</em>, 2024. <a href="https://www.biorxiv.org/content/10.1101/2024.01.16.575828v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/digital-microbe-a-genome-informed-data-integration-framework-for-collaborative-research-on-emerging-model-organisms-v2/">[PreLight]</a>

#### 2023 ####

<p>Patrick Maximilian Schwehn, Pascal Falter-Braun. Inferring protein from mRNA concentrations using convolutional neural networks. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.11.06.565778v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/inferring-protein-from-mrna-concentrations-using-convolutional-neural-networks-v3/">[PreLight]</a>

<p>Bipasha Dey, Verena Kaul, Girish Kale, Maily Scorcelletti, Michiko Takeda, Yu-Chiun Wang, Steffen Lemke. Divergent evolutionary strategies preempt tissue collision in fly gastrulation. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.10.09.561568v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/patterned-embryonic-invagination-evolved-in-response-to-mechanical-instability-divergent-evolutionary-strategies-preempt-tissue-collision-in-fly-gastrulation-final/">[PreLight]</a>

<p>Bruno C. Vellutini, Marina B. Cuenca, Abhijeet Krishna, Alicja Szałapak, Carl D. Modes, Pavel Tomančák. Patterned embryonic invagination evolved in response to mechanical instability. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.03.30.534554v2">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/patterned-embryonic-invagination-evolved-in-response-to-mechanical-instability-divergent-evolutionary-strategies-preempt-tissue-collision-in-fly-gastrulation-final/">[PreLight]</a>

<p>Keisuke Motone, Daphne Kontogiorgos-Heintz, Jasmine Wee, Kyoko Kurihara, Sangbeom Yang, Gwendolin Roote, Yishu Fang, Nicolas Cardozo, Jeff Nivala . Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.10.19.563182v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/multi-pass-single-molecule-nanopore-reading-of-long-protein-strands-with-single-amino-acid-sensitivity-v3/">[PreLight]</a>

<p>Nikolaos Meimetis, Douglas A. Lauffenburger, Avlant Nilsson. Inference of drug off-target effects on cellular signaling using Interactome-Based Deep Learning. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.10.08.561429v3">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/inference-of-drug-off-target-effects-on-cellular-signaling-using-interactome-based-deep-learning-v2/">[PreLight]</a>

<p>Jeanette A.I. Johnson, Genevieve L. Stein-O’Brien, et al. Digitize your Biology! Modeling multicellular systems through interpretable cell behavior. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.09.17.557982">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/digitize-your-biology-modeling-multicellular-systems-through-interpretable-cell-behavior-vers2/">[PreLight]</a>

<p>Song Feng, James A. Sanford, Thomas Weber, Chelsea M. Hutchinson-Bunch, Panshak P. Dakup, Vanessa L. Paurus, Kwame Attah, Herbert M. Sauro, Wei-Jun Qian, H. Steven Wiley. A Phosphoproteomics Data Resource for Systems-level Modeling of Kinase Signaling Networks. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.08.03.551714v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/a-phosphoproteomics-data-resource-for-systems-level-modeling-of-kinase-signaling-networks/">[PreLight]</a>

<p>Ian Smith, Petr Smirnov, Benjamin Haibe-Kains. Similarity metric learning on perturbational datasets improves functional identification of perturbations. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.06.09.544397v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/similarity-metric-learning-on-perturbational-datasets-improves-functional-identification-of-perturbations/">[PreLight]</a>

<p>Intekhab Hossain, Viola Fanfani, John Quackenbush, Rebekka Burkholz. Biologically informed NeuralODEs for genome-wide regulatory dynamics. <em>bioRxiv</em>, 2023. <a href="https://www.biorxiv.org/content/10.1101/2023.02.24.529835v1">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/biologically-informed-neuralodes-for-genome-wide-regulatory-dynamics/">[PreLight]</a>

<p>John D. Blair, Austin Hartman, Fides Zenk, Carol Dalgarno, Barbara Treutlein, Rahul Satija. Phospho-seq: Integrated, multi-modal profiling of intracellular protein dynamics in single cells. <em>bioRxiv</em>, 2023. <a href="https://doi.org/10.1101/2023.03.27.534442">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/phospho-seq-integrated-multi-modal-profiling-of-intracellular-protein-dynamics-in-single-cells/s">[PreLight]</a>

<p>Geena V. Ildefonso, Stacey D. Finley. A data-driven Boolean model explains memory subsets and evolution in CD8+ T cell exhaustion. <em>npj Syst Biol Appl</em>, 2023. <a href=" https://www.nature.com/articles/s41540-023-00297-2">[Paper]</a> <a href="https://prelights.biologists.com/highlights/a-data-driven-boolean-model-explains-memory-subsets-and-evolution-in-cd8-t-cell-exhaustion/">[PreLight]</a>

<p>Alexander Baryshev, Alyssa La Fleur, Benjamin Groves, Cirstyn Michel, David Baker, Ajasja Ljubetič, Georg Seelig. Massively parallel protein-protein interaction measurement by sequencing (MP3-seq) enables rapid screening of protein heterodimers. <em>bioRxiv</em>, 2023. <a href="https://doi.org/10.1101/2023.02.08.527770">[Preprint]</a> <a href="https://prelights.biologists.com/highlights/massively-parallel-protein-protein-interaction-measurement-by-sequencing-mp3-seq-enables-rapid-screening-of-protein-heterodimers/">[PreLight]</a>